submissions/EmmanuelIdiaghe/php-database/admin.php

```diff
@@ -0,0 +1,48 @@
+<?php
+include_once 'connect.php';
+$notSet = !(isset($_COOKIE["show"])); //cookie not set
+
+
+# Function that sets cookie
+function load() {
+    global $pdo;
+    $sql = "SELECT * FROM customers";
+    $result = $pdo->query($sql);
+    $details = $result->fetchAll();
+    setcookie("show", serialize($details), time() + (86400 * 30), "/");
+    $_COOKIE["show"] = serialize($details);
+}
+
+
+try {
+    if ($notSet) {
+        load(); //set cookie
+        echo "Please <b>REFRESH PAGE</b>";
```

Comment:
So, does that mean I have to tell the user to refresh the page to store data back into the cookie?

Reply:
Okay, I've taken care of that.
In order to avoid refreshing the page, I had to force the cookie to set immediately with $_COOKIE["show"]